x-pack/plugins/secops/public/components/event_details/helpers.tsx

```diff
@@ -75,46 +80,85 @@ interface GetItemsParams {
   populatedFields: EcsField[];
 }
 
+const DatesContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const DateFieldWithTooltip = pure<{ dateString: string }>(({ dateString }) => (
+  <EuiToolTip
+    data-test-subj="timeline-event-timestamp-tool-tip"
+    content={
+      <DatesContainer>
+        <FormattedRelative data-test-subj="last-updated-at-date" value={new Date(dateString)} />
+        <div>{moment(dateString).format('LLLL')}</div>
+        <div>{moment(dateString).format()}</div>
+      </DatesContainer>
+    }
+  >
+    <>{dateString}</>
+  </EuiToolTip>
+));
+
 /**
  * Given `data`, the runtime representation of an event,
  * and `populatedFields`, an `EcsField[]` containing all the fields that are
  * populated in `data`, it returns an `Item[]`, so the data can be shown in
  * the table
  */
-export const getItems = ({ data, populatedFields }: GetItemsParams): Item[] => {
-  return populatedFields.map((field, i) => ({
-    description: `${field.description} ${getExampleText(field)}`,
-    field: field.name,
-    type: field.type,
-    valueAsString: `${getMappedEcsValue({
-      data,
-      fieldName: field.name,
-    })}`,
-    value: (
-      <DraggableWrapper
-        key={`event-field-browser-value-for-${field.name}-${uuid.v4()}`}
-        dataProvider={{
-          enabled: true,
-          id: `id-event-field-browser-value-for-${field.name.replace('.', '_')}-${uuid.v4()}`, // escape '.'s in the field names
-          name: `${field.name}: ${getMappedEcsValue({
+export const getItems = ({ data, populatedFields }: GetItemsParams): Item[] =>
+  populatedFields.map(field => {
+    const itemDataProvider = {
+      enabled: true,
+      id: `id-event-field-browser-value-for-${field.name.replace('.', '_')}-${data._id!}`, // escape '.'s in the field names
```

Comment:
Just a heads up that there's an [`escapeDataProviderId(path: string)`](https://github.com/elastic/kibana/blob/43b02a38d50816e06e295155cd6d192e5c67eba0/x-pack/plugins/secops/public/components/drag_and_drop/helpers.ts#L59) helper floating around that DnD components are using